apps/src/templates/sectionProgress/standards/CreateStandardsReportStep2.jsx

```diff
@@ -117,6 +117,13 @@ const styles = {
   optionalText: {
     paddingLeft: 10,
   },
+  createButton: {
+    margin: 0,
+  },
+  backButton: {
+    margin: 0,
+    boxShadow: 'inset 0 2px 0 0 rgb(255 255 255 / 40%)',
```

Comment:
For my understanding, what does this little magic snippet do? Semi-relatedly, I know we don't have scss going here yet, but sharing this little mixin for future reference in case you haven't seen it (I heard at least one other developer this week say they hadn't, very helpful for removing default button styling!):

https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/mixins.scss

Reply:
That will be helpful for other cases. This one in particular wanted a box shadow up top. (see the slightly lighter color on the 'back' button)